apps/meteor/app/lib/server/functions/getAvatarSuggestionForUser.ts

```diff
@@ -53,7 +53,7 @@ const avatarProviders = {
 	},
 
 	twitter(user: IUser) {
-		if (user.services?.twitter?.profile_image_url_https && settings.get('Accounts_OAuth_Twitter')) {
+		if (user.services?.twitter?._image_url_https && settings.get('Accounts_OAuth_Twitter')) {
```

Comment:
not related. please remove this change

Reply:
Thank you for catching that. I have removed the unrelated changes, and the commit now only includes the intended updates for this issue.